Contentstack.Management.Core/Models/App.cs

```diff
@@ -134,7 +134,7 @@ public virtual ContentstackResponse FindAll(ParameterCollection collection = nul
         /// <pre><code>
         /// ContentstackClient client = new ContentstackClient(&quot;&lt;AUTHTOKEN&gt;&quot;, &quot;&lt;API_HOST&gt;&quot;);
         /// Organization organization = client.Organization(&quot;&lt;ORG_UID&gt;&quot;);
-        /// ContentstackResponse contentstackResponse = await organization.App(&quot;&lt;APP_UID&gt;&quot;).FindAllAsync(content);
+        /// ContentstackResponse contentstackResponse = await organization.App(&quot;&lt;APP_UID&gt;&quot;).FindAllAsync();
```

Comment:
previously it had content inside FindAllAsync();

Reply:
Example was wrong so updated it with removing content